src/js/sedambrojeva.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+const poljeZaIspisBrojeva = document.getElementById('brojevi')
+const dugme = document.getElementById('dugme')
+
+function generisiBrojeve () {
+  poljeZaIspisBrojeva.innerHTML = ' '
+  const brojevi = []
+  for (let i = 1; i < 37; i++) {
+    brojevi.push(i)
+  }
+  for (let i = brojevi.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1))
+    const temp = brojevi[i]
+    brojevi[i] = brojevi[j]
+    brojevi[j] = temp
+  }
+  for (let i = 0; i < 7; i++) {
+    poljeZaIspisBrojeva.innerHTML += brojevi.pop() + '<br>'
+  }
```

Comment:
Preradi funkciju da koristi return.

Reply:
Napravio sam novu komit sa svim izmenama. Da li treba da napravim i novi pull request za izmenjenu verziju?

Reply:
Ne to je to. Svi komiti idu na isti pull request dok ga ne sredimo.